src/main/kotlin/org/rust/ide/intentions/MutablyNamedIntention.kt

```diff
@@ -0,0 +1,14 @@
+package org.rust.ide.intentions
+
+import com.intellij.codeInsight.intention.PsiElementBaseIntentionAction
+import com.intellij.psi.PsiElement
+
+abstract class MutablyNamedIntention : PsiElementBaseIntentionAction() {
+    private var m_text = ""
+
+    protected abstract fun getTextForElement(element: PsiElement): String
+
+    override fun getText() = m_text
+    override fun getFamilyName() = text
```

Comment:
other intentions all use text as Family name so i just "stick" with the convention, and there is no family for this intention i think.